earth2mip/initial_conditions/ifs.py

```diff
@@ -50,13 +50,13 @@ def _get_channel(c: str, **kwargs) -> xarray.DataArray:
 def get(time: datetime.datetime, channels: List[str]):
```

Comment:
some updates, this feature has been broken for some time---since the test is xfailed, we didn't notice. I couldn't find a time to test against, so left the test as xfailed.